tools/alt-testing/README.md

```diff
@@ -0,0 +1,4 @@
+# This directory contains the scripts that can be used for testing Terraform and config-connector alternatives
+
```

Comment:
could you add a short setup section mentioning which tools are required? from a quick scan I saw

* `gcloud`
* `gsutil`
* `kubectl`
* `terraform`
* `kpt`